src/definitions/modules/nag.less

```diff
@@ -95,64 +95,116 @@ a.ui.nag {
            Variations
 *******************************/
 
+& when(@variationNagOverlay) {
+  /*--------------
+       Static
+  ---------------*/
 
-/*--------------
-     Static
----------------*/
-
-.ui.overlay.nag {
-  position: absolute;
-  display: block;
+  .ui.overlay.nag {
+    position: absolute;
+    display: block;
+  }
 }
 
+& when(@variationNagFixed) {
 /*--------------
      Fixed
 ---------------*/
 
-.ui.fixed.nag {
-  position: fixed;
+  .ui.fixed.nag {
+    position: fixed;
+  }
 }
-
+& when(@variationNagBottom) {
 /*--------------
      Bottom
 ---------------*/
-
-.ui.bottom.nags,
-.ui.bottom.nag {
-  border-radius: @bottomBorderRadius;
-  top: auto;
-  bottom: @bottom;
+  .ui.bottom.nags,
+  .ui.bottom.nag {
+    border-radius: @bottomBorderRadius;
+    top: auto;
+    bottom: @bottom;
+  }
 }
```

Comment:
Can we have all rounded borders if it's not fixed nag? Otherwise, it looks awkward as it's bottom borders are rounded, but the top borders are not.

Reply:
Yes, right 👍 
Did that by e1a940b97c2a26d64ff5d9fba340982ba5a58e62 and also supported fixed groups as well 🙂 
Updated screenshots and jsfiddle in de PR description.

Please review again